prime-router/start_func.sh

```diff
@@ -34,4 +34,4 @@ cat <<EOT >> local.settings.json
 }
 EOT
 fi
-func host start --language-worker -- "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```

Comment:
this change looks like it will prevent debugging the function running in a container.

Reply:
Checked in as a mistake

Reply:
Reverted